### PR TITLE
[Task] add Pest 

### PR DESCRIPTION
**Pull request by oallain**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes
| Fixed tickets | #811 

Add PestPHP https://pestphp.com to GrumPHP :rocket: 
Task done with :heart: for Pest.

# New Task Checklist:

- [x] Are the dependencies added to the composer.json suggestions?
- [x] Is the doc/tasks.md file updated?
- [x] Are the task parameters documented?
- [x] Is the task registered in the tasks.yml file?
- [x] Does the task contains phpunit tests?
- [x] Is the configuration having logical allowed types?
- [x] Does the task run in the correct context?
- [x] Is the `run()` method readable?
- [x] Is the `run()` method using the configuration correctly?
- [x] Are all CI services returning green?

Execution with an output error (for show): 

image

### CODE REVIEW

File: doc/tasks/pest.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+***Config***
+
+The task lives under the `pest` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+grumphp:
+    tasks:
+        pest:
```
Comment (veewee) [inline doc/tasks/pest.md:19]: Would it make sense to allow the same options as the phpunit task?

**Review by veewee — COMMENTED** [review]
Thanks for the fast PR!
It looks good. Maybe the list of options is a bit short?

File: doc/tasks/pest.md (hunk 1 above)
Comment (oallain) [reply doc/tasks/pest.md:19]: Hi,

The list of options provides by Pest is the same à PHPUnit, so yes, doing same here is a good idea :wink:

Comment (oallain) [reply doc/tasks/pest.md:19]: PHPUnit provide `testsuite` and `group`, so i added it :smiling_face_with_three_hearts:

**Comment by oallain** [comment]
Just close en reopen because of Travis bug :-1:

**Comment by veewee** [comment]
Thanks for your lovely PR :)

**Pull request merged**